test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Error.Tests.ps1

```diff
@@ -108,4 +108,16 @@ Describe 'Get-Error tests' -Tag CI {
         $err.PSObject.TypeNames | Should -Contain 'System.Exception'
         $err.PSObject.TypeNames | Should -Not -Contain 'System.Exception#PSExtendedError'
     }
+
+    It 'Get-Error will not modify the original error object' {
+        try {
+            1 / 0
+        }
+        catch {
+        }
```

Comment:
I know it doesn't really matter, but is there a reason you didn't just use `Write-Error` for this purpose? 😄

Reply:
Didn't consider that and this was the first thing I thought of.  At least now it's consistent within the test script :)

Although I also learned later that you can just write to `$Error` anyways.